javaharness/java/arcs/android/AndroidUiBroker.java

```diff
@@ -1,27 +1,22 @@
-package arcs.api;
+package arcs.android;
 
 import java.util.HashMap;
 import java.util.Map;
+
 import javax.inject.Inject;
 
-public class UiBrokerImpl implements UiBroker {
+import arcs.api.PortableJson;
+import arcs.api.UiBroker;
+import arcs.api.UiRenderer;
+
+public class AndroidUiBroker implements UiBroker {
```

Comment:
is there anything android specific in this implementation that it shouldn't be in `arcs.api`?

Reply:
Merged into one class.